packages/app-accounts/src/Forgetting.tsx

```diff
@@ -0,0 +1,79 @@
+// Copyright 2017-2018 @polkadot/app-accounts authors & contributors
+// This software may be modified and distributed under the terms
+// of the ISC license. See the LICENSE file for details.
+import { I18nProps } from '@polkadot/ui-app/types';
+
+import React from 'react';
+
+import Button from '@polkadot/ui-app/Button';
+import Modal from '@polkadot/ui-app/Modal';
+import translate from './translate';
+
+type Props = I18nProps & {
+  isOpen: boolean,
+  onClose: () => void,
+  doForget: () => void
+};
+
+class Forgetting extends React.PureComponent<Props> {
+  constructor (props: Props) {
+    super(props);
+  }
+
+  render () {
+    const { isOpen, style } = this.props;
+
+    if (!isOpen) {
+      return null;
+    }
+
+    return (
+      <Modal
+        className='staking--Nominating'
+        dimmer='inverted'
+        open
+        style={style}
+      >
+        {this.renderContent()}
+        {this.renderButtons()}
+      </Modal>
+    );
+  }
+
+  renderButtons () {
+    const { onClose, doForget, t } = this.props;
+
+    return (
+      <Modal.Actions>
+        <Button.Group>
+          <Button
+            isNegative
+            onClick={onClose}
+            text={t('forget.close', {
+              defaultValue: 'Cancel'
+            })}
+          />
+          <Button.Or />
+          <Button
+            isPrimary
+            onClick={doForget}
+            text='Delete'
```

Comment:
Small oversight :) Translatable.

Reply:
Workflow question: should I make these changes in this pull request, even though it's already been merged / closed?

Reply:
I should probably not have merged with that outstanding - however I marked "all-ok" at night, then tested "all-ok" the next morning ... Not the end of the world, that is a small fix, but it slipped through :)